gnd/src/main/res/xml/preferences.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2020 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<PreferenceScreen xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <PreferenceCategory
+    app:key="general_category"
+    app:title="General">
+
+    <SwitchPreferenceCompat
+      app:key="upload_media"
+      app:summary="Over Wi-Fi only"
+      app:title="Upload media" />
+
+    <SwitchPreferenceCompat
+      app:key="offline_areas"
+      app:summary="Over Wi-Fi only"
+      app:title="Offline areas download" />
+
+  </PreferenceCategory>
+
+  <PreferenceCategory
+    app:key="help_category"
+    app:title="Help">
+
+    <Preference
+      app:key="visit_website"
+      app:summary="@string/ground_website"
+      app:title="Visit website" />
+
+    <Preference
+      app:key="feedback"
+      app:summary="Report technical issues or suggest new features"
+      app:title="Send feedback" />
+
+  </PreferenceCategory>
+
+</PreferenceScreen>
```

Comment:
Does the autoformat remove final CRs? Maybe they should be there?

Reply:
I always forget about this. Thanks!
Maybe there is a checkstyle or pmd rule for it.